packages/testing/src/PHPUnit/AbstractRectorTestCase.php

```diff
@@ -378,7 +378,7 @@ private function configureEnabledRectors(EnabledRectorsProvider $enabledRectorsP
 
     private function configurePhpVersionFeatures(): void
     {
-        if ($this->getPhpVersion() === '') {
+        if ($this->getPhpVersion() === 0) {
```

Comment:
This might be a bit confusing.
This zero and one above should be implemented in a named local constant, e.g

```php
private const PHP_VERSION_UNDEFINED = 0;
```

Reply:
implemented.